packages/core/package.json

```diff
@@ -41,7 +41,7 @@
     "color": "^3.1.2",
     "extract-zip": "^1.7.0",
     "inquirer": "^7.0.4",
-    "jimp": "^0.9.8",
+    "sharp": "^0.23.2",
```

Comment:
We should also remove the dependency on `jimp` from the `packages/core/package.json`